### DISCUSSION TITLE
virtnest/import/import-ubuntu/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4424] [Announcements]
# virtnest/import/import-ubuntu/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/virtnest/import/import-ubuntu/

**Comment by panpan0000** [comment]
标题不太合理，应该是如何从VMWare导入传统虚拟机到云原生虚拟机平台。
并且能区分import/clone/migrate

**Comment by windsonsea** [comment]
Fixed in https://github.com/DaoCloud/DaoCloud-docs/pull/4462
